### ISSUE TITLE
Possible problem with Wheezy-Jessie upgrade commands

### ISSUE THREAD

**Issue by bobthechemist** [NicoHood/Arduino-IDE-for-Raspberry#2]
I ran in to a problem with upgrading as instructed.  I needed to also update `/etc/apt/sources.list.d/raspi.list` to include `deb http://archive.raspberrypi.org/debian/ jessie main ui`.  Before this change, I noticed that raspi-config was not properly updated and programs such as Mathematica would not install.

**Comment by NicoHood** [comment]
Thx for the report, but as you see the source here is old now. Jessie is official released now.
You might want to try this, even though its outdated as well (sadly)
https://github.com/arduino/Arduino/pull/3549

**Comment by bobthechemist** [comment]
I agree; however, your writeup for getting Arduino 1.6.3 working on the RPi is most straightforward I've seen thus far, even if it is outdated. Thanks for your contributions to the maker community.
